crates/mun_codegen/src/ir/body.rs

```diff
@@ -218,9 +217,19 @@ impl<'a, 'b, D: IrDatabase> BodyIrGenerator<'a, 'b, D> {
     fn gen_struct_alloc(
         &mut self,
         hir_struct: hir::Struct,
-        args: &[BasicValueEnum],
+        args: Vec<BasicValueEnum>,
```

Comment:
Why not a slice? That'd more universal.

Reply:
Because I want to consume the contents completely.